flink-runtime/src/main/java/org/apache/flink/runtime/state/StateBackend.java

```diff
@@ -231,6 +234,7 @@
 	 *
 	 * @param <K> The type of the keys by which the state is organized.
 	 *
+	 * @param cancelStreamRegistry
```

Comment:
I suggest to either complete all parameters of the java doc (better) or at least to remove the newly introduced `@param cancelStreamRegistry`. It gives a doc warning like this.

Reply:
Let me remove the parameter for this commit and add the full javadoc in later one to reduce rebase efforts.